src/main/java/org/openmrs/addonindex/scheduled/FetchDetailsToIndex.java

```diff
@@ -123,8 +123,9 @@ void getDetailsAndIndex(AddOnToIndex toIndex) throws Exception {
 	}
 	
 	void fetchExtraDetailsForEachVersion(AddOnToIndex toIndex, AddOnInfoAndVersions infoAndVersions) throws Exception {
+        boolean idAndPackageSet;
```

Comment:
Call this variable `alreadySetIdAndPackage`

And also, you can assign it in the same line, e.g. `boolean alreadySetIdAndPackage = false`